HedgeModManager/Languages/pt-PT.xaml

```diff
@@ -230,6 +230,12 @@
     <system:String x:Key="GameSonicForces">Sonic Forces</system:String>
     <system:String x:Key="GamePuyoPuyoTetris2">Puyo Puyo Tetris 2</system:String>
     <system:String x:Key="GameTokyo2020">Olympic Games Tokyo 2020</system:String>
-    <system:String x:Key="GameSonicColorsUltimate">Sonic Colours: Ultimate</system:String>
```

Comment:
This game, in Europe, is localised as "Sonic Colours" and not "Colors".

Reply:
Sorry, I copy-pasted the last few lines to just add in the new game strings and I guess that one came with it.